### DISCUSSION TITLE
Would you accept a PR to let the input field handle Home and End keys in useComboBox

### DISCUSSION THREAD

**Discussion by stefanprobst** [adobe/react-spectrum#2187] [Ideas]
i realize i have brought this up before, but since i have been asked again myself - would you accept a PR to let the input field handle Home and End keys in useComboBox (instead of Home and End moving the selection in the listbox popup)? a [similar `handleHomeEndKeys` option](https://material-ui.com/api/autocomplete/#props) exists in the MaterialUI Autocomplete

i think these would be all the necessary changes: https://github.com/stefanprobst/react-spectrum/commit/f62a28c5b2cbc3ed21c071dbbbd4dad23c053336

**Comment by LFDanLu** [comment]
I would need to ask the team how they feel about this change, but in the meantime you could simply override "Home" and "End" on the `inputProps.keyDown` returned by `useCombobox` right?

**Reply by devongovett** [reply]
IMO ideally Home and End should navigate the list box if an option is currently focused, and otherwise move the cursor.

**Reply by stefanprobst** [reply]
> in the meantime you could simply override "Home" and "End" on the inputProps.keyDown returned by useCombobox

thanks, yes that would work.

> Home and End should navigate the list box if an option is currently focused

could you clarify what it means for an option to be focused? i had assumed that focus would always have to remain on the `input` field?
